bang/magicollections.py

```diff
@@ -2636,6 +2636,18 @@ def to_fields(self, view, item, extra_fields=None, exclude_fields=None, *args, *
                     'image': item.member.square_image_url,
                 })
             extra_fields.append(('costume', member_field_params))
+        
+        chibis_set = item.owned_chibis.all()
+        if chibis_set:
+            extra_fields.append(('chibis', {
+                'icon': 'pictures',
+                'type': 'images',
+                'verbose_name': _('Chibi'),
+                'images': [{
+                    'value': chibi.image_url,
+                    'verbose_name': _('Chibi'),
+                } for chibi in chibis_set],
```

Comment:
Same as above, use `images_links` to allow users to download full size images.
Since it's the same logic, it could also be done in a separate function, but it hasn't really been done that way so far so that's up to you!